plugins/automap/src/automap.service.ts

```diff
@@ -317,7 +328,7 @@ export class AutomapService {
     plan: Flatfile.JobExecutionPlan
   ): boolean {
     return R.pipe(plan, (p) =>
-      p.fieldMapping?.every(
+      p.fieldMapping?.some(
```

Comment:
@carlbrugger if you see this: This makes the mapping focused on columns not the entire file. I think this change makes sense but want to confirm